specs/interop/predeploys.md

```diff
@@ -146,6 +150,7 @@ properties about the `_msg`.
 |-------------------|----------------------------------------------|
 | Address           | `0x4200000000000000000000000000000000000023` |
 | `MESSAGE_VERSION` | `uint256(0)`                                 |
+| `EXPIRY_WINDOW`   | `TBD`                                        |
```

Comment:
Can we define this as 2 hours? Not sure what the best time should be to be honest, we could always modify it in the future